sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -220,8 +220,9 @@ export class ApplicationControllerService {
    */
   processApplicationIncomeVerificationDetails(
     craIncomeVerification: CRAIncomeVerification[],
-  ): ApplicationIncomeVerification {
-    const incomeVerificationDetails = {} as ApplicationIncomeVerification;
+  ): ApplicationIncomeVerificationAPIOutDTO {
```

Comment:
mm.. I guess if it's not part of the controller DTO, then no need to change it to `APIOutDTO`. right devs?